api/src/main/java/jakarta/data/event/PostDeleteEvent.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2023,2024 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package jakarta.data.event;
+
+/**
+ * An event that occurs when an {@link jakarta.data.repository.Delete}
+ * lifecycle method is called, and after each record is deleted from the datastore.
```

Comment:
Adjust wording to avoid possible misinterpretation that this post event occurs both when Delete is called and after a record is deleted.
```suggestion
 * An event that occurs when a {@link jakarta.data.repository.Delete}
 * lifecycle method is called, after each record is deleted from the datastore.
```